src/pcapi/utils/requests.py

```diff
@@ -82,5 +82,6 @@ def __init__(self, *args, **kwargs):
         safe_adapter = HTTPAdapter(max_retries=safe_retry_strategy)
         unsafe_adapter = HTTPAdapter(max_retries=unsafe_retry_strategy)
         self.mount("https://www.demarches-simplifiees.fr", safe_adapter)
-        self.mount(settings.JOUVE_API_DOMAIN, safe_adapter)
-        self.mount("https://api.mailjet.com", unsafe_adapter)
+        if settings.JOUVE_API_DOMAIN:
+            self.mount(settings.JOUVE_API_DOMAIN, safe_adapter)
+            self.mount("https://api.mailjet.com", unsafe_adapter)
```

Comment:
Je ne suis pas sûr d'avoir compris ta réponse, Romain. Pour moi, la ligne relative à Mailjet devraiot être à l'extérieur du bloc :

```suggestion
        self.mount("https://api.mailjet.com", unsafe_adapter)
        if settings.JOUVE_API_DOMAIN:
            self.mount(settings.JOUVE_API_DOMAIN, safe_adapter)
```

Reply:
effectivement, j'ai fixé.